build.cmd

```diff
@@ -0,0 +1,14 @@
+@echo off
```

Comment:
Curious, are you talking about old versions of Windows or some pake requirement for a .cmd file?